src/lib/modernExtend.ts

```diff
@@ -118,7 +120,7 @@ export function onOff(args?: OnOffArgs): ModernExtend {
 
     const result: ModernExtend = {exposes, fromZigbee, toZigbee, isModernExtend: true};
     if (args.ota) result.ota = args.ota;
-    if (args.endpoints) {
+    if (args.endpoints && args.disableMutliEndpointOverride !== true) {
```

Comment:
Can you give me a use case for this option? Why just not leave out `args.endpoints` if you don't want the endpoints?

Reply:
Without it I can’t add modern onOff with multiple endpoints and another extend with endpoint at the same time. Because other extends require string endpoint name which requires endpoints to be defined. And this override brakes it because it’s only for onOff switches. If I add all endpoints here it will be confusing an not explicit as to what endpoints are specifically for onOff. Is it reasonable solution to this problem?